<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ItemCreation</class>
 <widget class="QDialog" name="ItemCreation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>410</width>
    <height>435</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ItemCreation</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>20</y>
     <width>95</width>
     <height>106</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="VLayout_1">
    <property name="spacing">
     <number>6</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QPushButton" name="okButton">
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="cancelButton">
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>261</width>
     <height>401</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="VLayout_2">
    <item>
     <widget class="QLabel" name="outputLabel">
      <property name="text">
       <string>text</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QFormLayout" name="FLayout_2_1">
      <item row="0" column="0">
       <widget class="QLabel" name="form_nameLabel">
        <property name="text">
         <string>Name</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="form_barcodeLabel">
        <property name="text">
         <string>Barcode</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="form_manufacturerLabel">
        <property name="text">
         <string>Manufacturer</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="form_providerLabel">
        <property name="text">
         <string>Provider</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="form_descriptionLabel">
        <property name="text">
         <string>Description</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="form_restockLabel">
        <property name="text">
         <string>Restock</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="form_categoryLabel">
        <property name="text">
         <string>Category</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="form_contactLabel">
        <property name="text">
         <string>Contact</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="form_nameEdit"/>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="form_barcodeEdit"/>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="form_manufacturerEdit"/>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="form_providerEdit"/>
      </item>
      <item row="4" column="1">
       <widget class="QTextEdit" name="form_descriptionEdit"/>
      </item>
      <item row="5" column="1">
       <widget class="QSpinBox" name="form_restockSpinbox"/>
      </item>
      <item row="6" column="1">
       <widget class="QComboBox" name="form_categoryCB"/>
      </item>
      <item row="7" column="1">
       <widget class="QComboBox" name="form_contactCB"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>ItemCreation</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>330</x>
     <y>66</y>
    </hint>
    <hint type="destinationlabel">
     <x>197</x>
     <y>125</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
